policies/ControllerAffinityNodeSelector/Makefile

```diff
@@ -0,0 +1,41 @@
+# POLICY_TYPE can be one of `gatekeeper`, `opa`, with different build process
+POLICY_TYPE ?= gatekeeper
+SOURCE_FILES := $(shell find . -type f -name '*.rego')
+
+VERSION ?= $(sed -n -e 's/^version\:\ \(.*\)/\1/p' artifacthub-pkg.yml)
+
+policy.wasm: $(SOURCE_FILES)
+ifeq ($(POLICY_TYPE), gatekeeper)
+	opa build -t wasm -e policy/violation -o bundle.tar.gz policy.rego
+else ifeq ($(POLICY_TYPE), opa)
+	opa build -t wasm -e policy/main utility/policy.rego -o bundle.tar.gz policy.rego
+else
+	@printf "Please assign POLICY_TYPE to either 'gatekeeper' or 'opa'\n"
+	exit 1
+endif
+	tar xvf bundle.tar.gz /policy.wasm
+	rm bundle.tar.gz
+	touch policy.wasm # opa creates the bundle with unix epoch timestamp, fix it
+
+.PHONY: test
+test:
+	opa test policy.rego tests/*.rego
+
+artifacthub-pkg.yml: metadata.yml
+	$(warning If you are updating the artifacthub-pkg.yml file for a release, \
+		remember to set the VERSION variable with the proper value. \
+		To use the latest tag, use the following command:  \
+		make VERSION=$$(git describe --tags --abbrev=0 | sed 's/\(.*\)\/\(.*\)$/\2/' | cut -c2-) annotated-policy.wasm)
```

Comment:
Here we should suggest to run `Make VERSION artifacthub-pkg.yml` (instead of `annotated-policy.wasm`).

Since we should change this for all policies, could we copy-paste this Makefile for all policies?
https://github.com/viccuad/rego-policies/blob/main/policies/ControllerContainerBlockSSHPort/Makefile#L28-L29

Reply:
fixed